CondFormats/Common/interface/BasicPayload.h

```diff
@@ -9,14 +9,28 @@ namespace cond {
   */
   class BasicPayload {
   public:
-    
-    BasicPayload():m_data0(0.),m_data1(0.){}
-    BasicPayload( float d0, float d1):m_data0(d0),m_data1(d1){}
+    static const size_t vecSize = 100; 
+    BasicPayload():m_data0(0.),m_data1(0.),m_vec(vecSize,0){
+    }
+    BasicPayload( float d0, float d1):m_data0(d0),m_data1(d1),m_vec(vecSize,0){
+      for(size_t i=0;i<vecSize;i++) m_vec[i] = d0*i+d1;
+    }
     virtual ~BasicPayload(){}
+
+    void print() {
+      for ( size_t i=0; i<10; i++ ) {
+	for ( size_t j=0;j<10;j++ ){
+	  size_t ind = i*10+j;
+	  std::cout <<ind<<":"<<m_vec[ind]<<"  ";
+	}
+	std::cout <<std::endl;
+      }
+    }
     
   public:
     float m_data0;
     float m_data1;
+    std::vector<float> m_vec;
```

Comment:
@davidlange6 
Ok, point taken. This was only an example of simple payload. The fixed sized has been relaxed.